its/core-it-suite/src/test/resources/mng-3694/maven-mng3694-plugin/src/main/java/plugin/MyMojo.java

```diff
@@ -47,24 +47,12 @@
  */
 public class MyMojo extends AbstractMojo {
 
-    /**
```

Comment:
I think the comment shows why and how this is used here. Please check this sort of thing before sending PRs from automated tools.

Reply:
yes im sorry.

order 100% test coverage or deletion.